src/main/java/seedu/address/model/ItemModel.java

```diff
@@ -1,102 +1,65 @@
 package seedu.address.model;
 
-import seedu.address.model.item.EventList;
+import java.nio.file.Path;
+
+import seedu.address.commons.core.GuiSettings;
 import seedu.address.model.item.Item;
 import seedu.address.model.item.ItemList;
-import seedu.address.model.item.ReminderList;
-import seedu.address.model.item.TaskList;
```

Comment:
Hi! Bryan has changed these to seedu.address.commons.core.item... perhaps you can change it too to avoid future bugs:)

Reply:
okay! will edit it when his request have been merged.